hypha/apply/determinations/views.py

```diff
@@ -279,6 +279,7 @@ def get_object(self, queryset=None):
         return self.model.objects.get(submission=self.submission, is_draft=True)
 
     def dispatch(self, request, *args, **kwargs):
+
```

Comment:
Unrelated whitespace change.  No big deal, and certainly not worth holding anything up about, but in general those are good to avoid in a code diff (at least IMHO).